Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.yml

```diff
@@ -57,15 +49,17 @@ configuration:
   name: key
   required: false
   type: 14
-- additionalinfo: Ignores certificate and private key, and will run the export indicators service
-    in HTTP
-  display: HTTP Server
-  hidden: false
+- display: HTTP Server
   name: http_flag
   required: false
   type: 8
-description: Use the Export Indicators Service integration to provide an endpoint with a list of indicators as a service for
-  the system indicators.
+- display: Username
+  additionalinfo: Set the username which the user should enter when entering the indicators page.
```

Comment:
not sure if its a good description, lets work with @kirbles19  on that

Reply:
See my suggestion on EDL